queenbee/cli/config/__init__.py

```diff
@@ -0,0 +1,23 @@
+from .view import view
+from .auth import auth
+
+try:
+    import click
+except ImportError:
+    raise ImportError(
+        'click modules not installed. Try `pip install queenbee[cli]` command.'
+    )
+
+
+@click.group('config')
+@click.pass_context
+def main(ctx):
+    """manage queenbee configuration data
+    
+
+    """
```

Comment:
Are you trying to test my PEP8 flexibility? 😀

```suggestion
    """manage queenbee configuration data."""
```

Reply:
My bad... I... have no excuses at this point...